File: SPOJ/CANDY3.cpp
#include <bits/stdc++.h>
#define gi(x) scanf("%llu",&x)
using namespace std;
int main() {
	unsigned long long t,n,sum=0,a;
	gi(t);
	while(t--){
		gi(n);
		sum=0;
		for(long long i=0;i<n;i++){
			gi(a);
			sum+=a;
		}
		if(sum%n){
			printf("NO\n");
		}
		else{
			printf("YES\n");
		}

	}
	return 0;
}

File: SPOJ/CANDY3.py
t=raw_input()
t=(int)(t)
for test in xrange(t):
	n=raw_input()
	n=raw_input()
	n=(int)(n)
	#print n
	sum=0
	for i in xrange(n):
		a=raw_input()
		a=(int)(a)
		sum+=a
	if(sum%n):
		print "NO"
	else:
		print "YES"
		
